utils/tmpcomp

```diff
@@ -84,6 +84,13 @@ ${prefix}/src/stklos -c -q -b ${prefix}/src/boot.img \
      -f ${prefix}/utils/stklos-compile.stk -- \
      --no-time $Copt --output=$out $in
 
+status=$?
+if test $status -gt 0
+then
+    rm -f "$out"
+    exit $status
+fi
+
```

Comment:
I wanted to write:

```suggestion
     --no-time $Copt --output=$out $in || exit $?
```

but that left behind a 0-byte file which was enough to make `make` think the target had been successfully built.

It would be better if `stklos` did not leave behind an empty (or in any other ways incorrect) file when compilation fails but I didn't know how to do that.